docs/modules/ROOT/pages/migration-guide/3.0.0.adoc

```diff
@@ -0,0 +1,32 @@
+= Camel Quarkus 3.0.0 Migration Guide
+
+The following guide outlines how to adapt your code to changes that were made in Camel Quarkus 3.0.0.
+
+== Changes in Quarkus
+
+Please refer to xref:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0[Quarkus 3.0.0 Migration Guide].
+
+== Changes in Camel
+
+Please refer to xref:https://camel.apache.org/manual/camel-4-migration-guide.html[Apache Camel 3.x to 4.0 Migration guide].
+
+== Removed extensions
+
+The following extensions have been removed.
+
+[options="header"]
+|===
+| Extension | Alternative extension(s)
+| camel-quarkus-activemq               | camel-quarkus-jms, camel-quarkus-sjms, camel-quarkus-amqp
+| camel-quarkus-caffeine-lrucache      | camel-quarkus-ignite, camel-quarkus-infinispan
+| camel-quarkus-elasticsearch-rest     | TODO Coming camel-quarkus-elasticsearch
+| camel-quarkus-johnzon                | camel-quarkus-jackson, camel-quarkus-fastjson, camel-quarkus-gson
+| camel-quarkus-microprofile-metrics   | camel-quarkus-micrometer, camel-quarkus-opentelemtry
+| camel-quarkus-opentracing            | camel-quarkus-micrometer, camel-quarkus-opentelemtry
+| camel-quarkus-optaplanner            | none
+| camel-quarkus-rabbitmq               | none
+| camel-quarkus-vm                     | camel-quarkus-seda
+| camel-quarkus-xstream                | camel-quarkus-jacksonxml
```

Comment:
There is more: 
Atmos - removed from Camel
Avro RPC https://github.com/apache/camel-quarkus/issues/4462
DataSonnet - should come back https://github.com/apache/camel-quarkus/issues/4685
Dozer - missing in Camel 4, altough https://camel.apache.org/manual/camel-4-migration-guide.html does not say so
HBase - removed from Camel
Iota - removed from Camel
JBPM  - removed from Camel
JClouds  - removed from Camel
Milo - removed from Camel
smallrye-reactive-messaging - will come back https://github.com/apache/camel-quarkus/issues/4637
Tika - should come back https://github.com/apache/camel-quarkus/issues/4470
XML Security Sign - will come back https://github.com/apache/camel-quarkus/issues/4684

Reply:
Sorry, now I noticed this is a bakcport. The same applies to `main`

Reply:
Ok then let me add those to main, and port back here

Reply:
The alternative extensions column is great :+1: